piptools/scripts/options.py

```diff
@@ -364,3 +370,28 @@ def _get_default_option(option_name: str) -> Any:
     is_flag=True,
     help="Restrict attention to user directory",
 )
+
+build_deps_for = click.option(
+    "--build-deps-for",
+    "build_deps_for_distributions",
+    multiple=True,
+    type=click.Choice(ALL_BUILD_DISTRIBUTIONS),
+    help="Name of a distribution to install build dependencies for; may be used more than once. "
+    "Static dependencies declared in 'pyproject.toml::build-system.requires' will be included as "
+    "well.",
+)
+
+all_build_deps = click.option(
+    "--all-build-deps",
+    is_flag=True,
+    default=False,
+    help="Install build dependencies for all distributions. "
+    "Static dependencies declared in pyproject.toml will be included as well.",
+)
+
+build_deps_only = click.option(
+    "--build-deps-only",
```

Comment:
```suggestion
    "--only-build-deps",
```
Maybe but `only` in front? To be consistent with `--all-build-deps` and `pip install --only-binary`.

Reply:
Will fix